src/framed/std/core.cljc

```diff
@@ -121,10 +122,11 @@
   ([f y x]
    (f x y)))
 
-(defmacro future-loop
-  "Execute body repeatedly within a future, returning the future"
-  [& body]
-  `(future
-     (loop []
-       ~@body
-       (recur))))
+#?(:clj
+  (defmacro future-loop
```

Comment:
I would think that would work out of the box? but to piggyback on this, I wonder if macro definitions need to scoped to clj-only ..?